src/pages/graphql/schema/gift-registry/mutations/create.md

```diff
@@ -12,6 +12,8 @@ This mutation requires a valid [customer authentication token](../../customer/mu
 
 The `id` input attribute is optional. If a value is not specified, the application creates one. If you specify a value, then you can create a gift registry and make multiple updates in a single call.
 
+The `dynamic_attributes` input field contains an array of attributes that define elements of the gift registry. Each attribute is specified as a code-value pair
```

Comment:
```suggestion
The `dynamic_attributes` input field contains an array of attributes that define elements of the gift registry. Each attribute is specified as a code-value pair.
```

Reply:
Do you mean "key-value pair"? I've never seen "code-value" before, but that doesn't mean it's not correct as-is.